### ISSUE TITLE
Readme misleading regarding version in this repo?

### ISSUE THREAD

**Issue by TechupBusiness** [openaudible/openaudible#212]
Dear OpenAudible Team,

when someone reads your Readme, it seems that it's not the latest version in this repo:

```
# Note
This open source version 1.1.8 is not the latest available as a binary version. You can still build this version from scratch, but it will not be the latest version available.
```
and
```
# Building version 1.1.8
...
```

But obviously there are tags that match the latest binary from your website. So I guess it is indeed the latest version available. I would recommend to change the readme text.

**Comment by openaudible** [comment]
Not sure what you want the read me to say. We aren't releasing the latest source code any longer. Too much work, no useful code additions were ever submitted, and the basic functionality is in 1.1.8.

**Comment by TechupBusiness** [comment]
Hm. So this repo does not contain the latest source? Why do you have a git tag v1.5.4 then? 
I don't see why committing to git is much work ;-) And contributions usually does not come in a reliable steady way.

Maybe you should change the description on the website then, if it's not really opensource any longer. There you state: `An open-source cross-platform audible audiobook manager.` which seems to be true only for 1.1.8 but not the most current versions. Whats the difference between the open source 1.1.8 and your closed source 1.5.4? :thinking:

**Comment by anishmittal2020** [comment]
@openaudible Please stop releasing closed source binary when the code is not update to the latest version. 

This is really misleading!
